include/fizzy/fizzy.h

```diff
@@ -11,8 +11,87 @@
 extern "C" {
 #endif
 
+/// The opaque data type representing a module.
+struct fizzy_module;
```

Comment:
I'd prefer to name types as `FizzyModule`. In any case, it is good to make the decision now.

Reply:
Yeah, I'm thinking now, there's not much sense in having very different naming in C++ and C.

Reply:
My preference is `CamelCase` for types, `snake_case` for functions and variables, as in C++.

Reply:
> My preference is `CamelCase` for types, `snake_case` for functions and variables, as in C++.

Okay with that for now, Rust follows the same too, at least a lot of the code will be consistent.